OpenRA.Game/Renderer.cs

```diff
@@ -79,17 +77,6 @@ static Size GetResolution(GraphicSettings graphicsSettings)
 			return new Size(size.X, size.Y);
 		}
 
-		static IGraphicsDevice CreateDevice(Assembly platformDll, int width, int height, WindowMode window)
-		{
-			foreach (PlatformAttribute r in platformDll.GetCustomAttributes(typeof(PlatformAttribute), false))
```

Comment:
You can remove the `System.Reflection` using from both `Renderer.cs` and `Sound.cs` now.